catalog/src/main/kotlin/com/adevinta/spark/catalog/examples/samples/rating/RatingExamples.kt

```diff
@@ -82,13 +80,6 @@ public val RatingExamples: List<Example> = listOf(
     ) {
         RatingSimpleSample()
     },
-    Example(
```

Comment:
Rating star is a component used by our consumers we need to provide this example